rosetta-source/src/main/java/cdm/product/template/processor/FraPayoutSplitterMappingProcessor.java

```diff
@@ -107,10 +107,10 @@ private void addFloatingLegQuantityReference(Mapping mapping, InterestRatePayout
 
 	private void updateFloatingRateIndexReference(Mapping mapping, InterestRatePayoutBuilder floatingLeg) {
 		Reference.ReferenceBuilder reference = Optional.of(floatingLeg)
-				.map(InterestRatePayoutBuilder::getRateSpecification)
-				.map(RateSpecificationBuilder::getFloatingRate)
-				.map(FloatingRateBuilder::getRateOption)
-				.map(ReferenceWithMetaFloatingRateOptionBuilder::getReference)
+				.map(b -> b.getRateSpecification())
```

Comment:
These are no-op changes, but the previous code does not compile is certain java compilers